src/plugins/intel_cpu/src/nodes/reduce.cpp

```diff
@@ -3061,6 +3064,10 @@ inline void Reduce::set_reduce_dim_flags() {
         SET_DST_DIM_VALUE(1, process_dst_dims[0], 1, 1, 1);
     }
 
+    // Depthwise fusion can be computed like eltwise fusion without broadcast, if is_depthwise_compatible is true.
+    bool is_depthwise_compatible = dims_size > 1 && process_dst_dims[1] == OB * OC * OD * OH * OW;
```

Comment:
In typical scenario, seems like if spatial size(OD x OH x OW) is 1, we should load(store) across channels for ncsp layout for vectorization, in this case broadcast is false. Why we always have limitation on OB (I think reduce on batch is a rare case, so should in parallel domain)?

Reply:
Good point! Applied. Thanks Chenhu!

Reply:
need condition "reduce axis is not on batch"?